src/xml/inputdevicexml.cpp

```diff
@@ -295,7 +295,7 @@ void InputDeviceXml::readConfig(QXmlStreamReader *xml)
                         }
                     } else if ((xml->name() == "sensorname") && xml->isStartElement())
                     {
-                        int type = xml->attributes().value("type").toString().toInt();
+                        int type;
```

Comment:
Could you remove changes in this file?   
In these two particular cases our Linter seems to be wrong.  
These values are clearly used. We just map these int-s to enums